### ISSUE TITLE
textStyle1 styles not been applied

### ISSUE THREAD

**Issue by rick427** [calintamas/react-native-toast-message#233]
Hi, so i tried changing the style of the text1 prop like so:

```js
error: ({ text1, text2, ...rest }) => (
        <BaseToast
            {...rest}
            leadingIcon={ICONS.error}
            style={{ borderLeftColor: COLORS.danger }}
            text1Style={{
                ...FONTS.h6
            }}
            text2Style={{
                ...FONTS.body5
            }}
            text1={text1}
            text2={text2}
        />
    ),
```

Problem is that the styles in FONTS.h6 i.e ( fontFamily: 'Poppins-Regular', fontSize: 12 ) are not been applied. What am i doing wrong pls ?

**Comment by max-frai** [comment]
Same for me, the styling is not applied.

**Comment by mghozynAGIT** [comment]
Same for me, the styling in text1 is not applied.
It only works after trying insert fontWeight: 'normal' in text1Style

**Comment by calintamas** [comment]
Fixed in `v2.0.0`. [Read the complete changelog](https://github.com/calintamas/react-native-toast-message/releases/tag/v2.0.0). 

If you find any issues with the v2 implementation, feel free to reopen this issue. Thanks!
